tests/test_builder.py

```diff
@@ -0,0 +1,37 @@
+from unittest import TestCase
+
+from apluslms_roman.builder import Builder
+from apluslms_roman.configuration import ProjectConfig
+
+
+class TestBuilder(TestCase):
+
+    def setUp(self):
+        config = {
+            'version': '2.0',
+            'steps': [
+                {'img': 'a', 'name': 'test1'},
+                {'img': 'b', 'name': 'test2'},
+                {'img': 'c'}
+            ]}
+        config = ProjectConfig(ProjectConfig.Container(
+            '/a', allow_missing=True), None, config, None)
+        self.builder = Builder(None, config)
+
+    def test_get_steps(self):
+        steps = self.builder.get_steps()
+        self.assertEqual(len(steps), 3)
+        names = [s.name for s in steps]
+        self.assertEqual(['test1', 'test2', None], names)
```

Comment:
It's mgiht be good to switch actual and expected parameters around, just to keep consistency between tests.